### PR TITLE
drop type specs in RecoBTag

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description: 
Update the safer syntax for existing parameter :
- drop type specifications where the original parameter exists.
- move all parameter inside the clone

Instead of modifying parameters with full type specs, which can be interpreted as an insertion of a new parameter, it is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
(The previous PR were [PR#31162](https://github.com/cms-sw/cmssw/pull/31162),[PR#31243](https://github.com/cms-sw/cmssw/pull/31243),[PR#31332](https://github.com/cms-sw/cmssw/pull/31332), [PR#31389](https://github.com/cms-sw/cmssw/pull/31389), [PR#31523](https://github.com/cms-sw/cmssw/pull/31523))

In this PR, total 41 files changed. 
- RecoBTag/CTagging  : 6 files 
- RecoBTag/Combined : 3 files
- RecoBTag/ImpactParameter :  15 files 
- RecoBTag/ONNXRuntime : 1 file
- RecoBTag/SecondaryVertex : 16 files

#### PR validation:
Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_11_2_X, the basic test all passed in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: RecoBTag/CTagging/python/charmTagsComputerCvsL_cfi.py [hunk 1]
```diff
@@ -34,6 +34,8 @@
 charmTagsComputerCvsL.slComputerCfg.correctVertexMass = False
 
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
-phase1Pixel.toModify(charmTagsComputerCvsL, weightFile = cms.FileInPath('RecoBTag/CTagging/data/c_vs_udsg_PhaseI.xml'))
-phase1Pixel.toModify(charmTagsComputerCvsL, slComputerCfg = dict(correctVertexMass = True))
-phase1Pixel.toModify(charmTagsComputerCvsL, defaultValueNoTracks = cms.bool(True))
+phase1Pixel.toModify(charmTagsComputerCvsL, 
+                     weightFile = cms.FileInPath('RecoBTag/CTagging/data/c_vs_udsg_PhaseI.xml'),
```
Comment (perrotta) [inline RecoBTag/CTagging/python/charmTagsComputerCvsL_cfi.py:38]: What happens if you also drop the `FileInPath` type specifiers?

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Configurations cleaning implemented as anticipated in the PR descriptions
- Jenkins tests pass and show no differences, as expected

**Comment by silviodonato** [comment]
+1

File: RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py [hunk 2]
```diff
@@ -12,13 +12,17 @@
 	)
 
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
-phase1Pixel.toModify(pfDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
-phase1Pixel.toModify(pfDeepCSVJetTags, checkSVForDefaults = cms.bool(True))
-phase1Pixel.toModify(pfDeepCSVJetTags, meanPadding = cms.bool(True))
-phase1Pixel.toModify(pfDeepCSVJetTags, toAdd = cms.PSet())
+phase1Pixel.toModify(pfDeepCSVJetTags, 
+                     NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'),
```
Comment (silviodonato) [inline RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py:16]: if `NNConfig` already exists, you can drop also `cms.FileInPath`

Comment (perrotta) [reply RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py:16]: > if `NNConfig` already exists, you can drop also `cms.FileInPath`

Indeed, thank you @silviodonato 
The same at L24 below
We'll submit a quick patch as soon as this PR gets included in a IB

Comment (perrotta) [reply RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py:16]: Finalization in #31568 (submitted now before forgetting it)

**Pull request merged**